recipes/_libffi.rb

```diff
@@ -0,0 +1,48 @@
+#
+# Cookbook Name:: omnibus
+# Recipe:: _libffi
+#
+# Copyright 2015, Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# We need libffi 3.2.1 on ppc64/ppc64le systems in order
+# for ruby ffi gem to get installed. Someday when ruby ffi
+# updates embedded libffi 3.2.1 and fixes its extension
+# build process to consume embedded libffi properly this wont
+# be required. https://github.com/ffi/ffi/pull/424#issuecomment-83935000
+
+# Include the common recipe
+include_recipe 'omnibus::_common'
+
+include_recipe 'omnibus::_compile'
+
+chef_gem "pkg-config"
+require "pkg-config"
+
+# libffi.pc is installed at below path and its not default
+# search paths for pkg-config
+ENV['PKG_CONFIG_PATH'] = '/usr/local/lib/pkgconfig/'

```

Comment:
It might be cleaner to add an `environment` attribute to the `remote_install` resource. I just did a similar thing for the `ruby_install` resource:
https://github.com/opscode-cookbooks/omnibus/commit/598428870a1301ecc084b69ff6cfe278be7403b4
https://github.com/opscode-cookbooks/omnibus/commit/cf012a06572fa6d2a26ce90aaac7db2e9a157ada